rails/locale/ko.yml

```diff
@@ -154,7 +154,7 @@ ko:
         separator: "."
         significant: false
         strip_insignificant_zeros: false
-        unit: 원
+        unit: "₩"
```

Comment:
If there is a change here, it is necessary to change L152
```L152
format: "%u%n"
```

ex) 
1000원 -> ₩1000

Reply:
It's line 126 that needs to change @ryush00

Reply:
`1000원`
`₩1000`
both are correct in Korean grammar. (ref: https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=11018)

In English, $ is longer (dollar) so it doesn't commonly used, but in Korean, 원 is short, so it more commonly used.
So I think `1000원` is more fit to Korean translation.

Reply:
For Rails apps we should prefer symbols like $ ¥ € ₩ etc. So let's go with:

```
format: "%u%n"
unit: "₩"
```

Reply:
You're right. I'm looking at Naver, Coupang, SSG and they do all tend to use 원 instead of ₩. So let's revert the change and go with:

```
unit: 원
format: "%n%u"
```

Reply:
![image](https://user-images.githubusercontent.com/27655/195155509-bc80cce4-0bff-4040-8e16-71cf278bb903.png)

Reply:
```suggestion
        unit: 원
```